### PR TITLE
VLAN Connect and Network Show

### PR DESCRIPTION
**Pull request by uzaxirr**
Depends on: https://github.com/civo/civogo/pull/191

### CODE REVIEW

**Comment by uzaxirr** [comment]
Screenshot 2024-04-09 at 7 52 50 AM

**Comment by uzaxirr** [comment]
Screenshot 2024-04-09 at 7 53 56 AM

File: cmd/network/network.go [hunk 1]
```diff
@@ -25,7 +25,17 @@ func init() {
...
+	NetworkCmd.AddCommand(networkShowCmd)
+	NetworkCmd.AddCommand(networkConnectCmd)
 
 	networkCreateCmd.Flags().StringVarP(&cidrV4, "cidr-v4", "", "", "Custom IPv4 CIDR")
 	networkCreateCmd.Flags().StringSliceVarP(&nameserversV4, "nameservers-v4", "", nil, "Custom list of IPv4 nameservers (up to three, comma-separated)")
+
+	// Flag binding for networkConnectCmd
+	networkConnectCmd.Flags().IntVar(&vlanID, "vlan-id", 0, "VLAN ID to connect")
+	networkConnectCmd.Flags().StringVar(&vlanCIDRV4, "cidr-v4", "", "CIDR v4 of the VLAN")
+	networkConnectCmd.Flags().StringVar(&vlanGatewayIPv4, "gateway-ip-v4", "", "Gateway IPv4 address for the VLAN")
```
Comment (alessandroargentieri) [inline cmd/network/network.go]: can you change it into `gateway-ipv4` so it's the same as the JSON `gateway_ipv4`

Comment (uzaxirr) [reply cmd/network/network.go]: https://github.com/civo/cli/pull/379/commits/b9676dc5ba8794dc277c1cc1d5e7383141b3bbad

**Review by alessandroargentieri — APPROVED** [review]
LGTM

**Review by vishalanarase — APPROVED** [review]

**Pull request merged**